Recipes/Apply_function_to_every_gridpoint.ipynb

```diff
@@ -37,12 +37,13 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    sst = cat[experiment].search(frequency="1mon", variable="sst").to_dask()
</pre><p>Technicallly unrelated to this review but I think this would be faster if we open larger chunks in x &amp; y</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/461/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I'm seeing a small speed-up by not chunking `xt_ocean` and `yt_ocean` at open, and then chunking down when I tie all the time data together in the next cell, but really not all that much.

Reply:
Oh interesting - ok thanks :)

Reply:
I think the limiting factor is the number of `open_dataset` calls that it has to make - the total dataset is about 4GB all told, so the post-open handling isn't particularly taxing. This sort of chunking may have more effect with really big datasets.

Reply:
I think specifying larger chunks  as `xarray_open_kwargs` reduces the number of `open_dataset` calls? I assume it does one `open_dataset` call per chunk, so the minimum number of calls is achieved through one chunk per file ?

Reply:
<p>See I thought it just does one <code>open_dataset</code> call per file... might check that </p>

Reply:
Is this something we working on in this PR? If not, then happy to merge from me! :)

Reply:
No this is resolved